assets/js/video.js

```diff
@@ -0,0 +1,83 @@
+import {Peer} from "peerjs"
```

Comment:
Could you add a line to the README, that we have to:

```
npm install peerjs --save
```

while in `assets/` to avoid

```
[ERROR] Could not resolve "peerjs"
```

when running the project after a clean checkout?

Reply:
@rafalskorupa Could you take a look at this?

Reply:
Ah, ok, I see it being addressed in the other PRs